src/error/error-qvm.lisp

```diff
@@ -0,0 +1,142 @@
+;;;; error-qvm.lisp
+;;;;
+;;;; This file implements an efficient QVM which tracks the propagation of Pauli
+;;;; errors through a CNOT-dihedral circuit which, in the absence of errors, is
```

Comment:
little disappointed there's not a 3D ascii art

Reply:
💎